wishlist.html

```diff
@@ -0,0 +1,92 @@
+            <a href="/cart.html" class="nav-bar-icon"><i class="fa fa-cart-plus" aria-hidden="true"></i></a>
+            <a  href="#" class="nav-bar-icon"><i class="fa fa-user" aria-hidden="true"></i></a>
+        </div>
+    </nav>
+    
+    <main class="main">
+        <h2 class="wishlist-title">My Wishlist</h2>
+        <div class="main-body">
+            <div class="card">
+                <div class="image">
+                    <img src="/assets/sofa-1.jpeg" alt="card-image">
+                </div>
+                <div class="card-content">
+                    <div class="card-title">Sofa</div>
+                    <div class="card-sub-title">Rs. 4000</div>
+                    <div class="footer flex flex-center">
+                            <div class="product-card-button">
+                                <button class="card-button-cart">Go To Cart</button>
+                                <button class="card-button-wishlist">Remove From Wishlist</button>
+                            </div>
+                    </div>
+                </div>
+            </div>
+            <div class="card">
+                <div class="image">
+                    <img src="/assets/sofa-1.jpeg" alt="card-image">
+                </div>
+                <div class="card-content">
+                    <div class="card-title">Sofa</div>
+                    <div class="card-sub-title">Rs. 4000</div>
+                    <div class="footer flex flex-center">
+                            <div class="product-card-button">
+                                <button class="card-button-cart">Go To Cart</button>
+                                <button class="card-button-wishlist">Remove From Wishlist</button>
+                            </div>
+                    </div>
+                </div>
+            </div>
+            <div class="card">
+                <div class="image">
+                    <img src="/assets/sofa-1.jpeg" alt="card-image">
+                </div>
+                <div class="card-content">
+                    <div class="card-title">Sofa</div>
+                    <div class="card-sub-title">Rs. 4000</div>
+                    <div class="footer flex flex-center">
+                            <div class="product-card-button">
+                                <button class="card-button-cart">Go To Cart</button>
+                                <button class="card-button-wishlist">Remove From Wishlist</button>
+                            </div>
+                    </div>
+                </div>
+            </div>
+            <div class="card">
+                <div class="image">
+                    <img src="/assets/sofa-1.jpeg" alt="card-image">
+                </div>
+                <div class="card-content">
+                    <div class="card-title">Sofa</div>
+                    <div class="card-sub-title">Rs. 4000</div>
+                    <div class="footer flex flex-center">
```

Comment:
I see that there is in fact a utility class used here for setting display to flex. Try to use this in other components above as well.